blocktopmenu.php

```diff
@@ -52,7 +52,7 @@ public function __construct()
     {
         $this->name = 'blocktopmenu';
         $this->tab = 'front_office_features';
-        $this->version = '2.2.2';
+        $this->version = '2.3';
```

Comment:
On Fri, Aug 07, 2015 at 02:31:25AM -0700, Gregory Roussac wrote:
> > @@ -52,7 +52,7 @@ public function __construct()
> >      {
> >          $this->name = 'blocktopmenu';
> >          $this->tab = 'front_office_features';
> > -        $this->version = '2.2.2';
> > +        $this->version = '2.3';

ok.

But how to name the update function then ? It should take the name of
the next version it will apply to.